src/useStyles.js

```diff
@@ -0,0 +1,55 @@
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = (borderOptions) => makeStyles((theme) => {
+  const containerBorderStyle = borderOptions.isBorder === true && {
+    borderWidth: `${borderOptions.borderStyles.borderWidth ? borderOptions.borderStyles.borderWidth : 1}px`,
+    borderStyle: `${borderOptions.borderStyles.borderStyle ? borderOptions.borderStyles.borderStyle : 'solid'}`,
+    borderColor: `${borderOptions.borderStyles.borderColor ? borderOptions.borderStyles.borderColor : '#e0e0e0'}`,
+    borderRadius: `${borderOptions.borderStyles.borderRadius ? borderOptions.borderStyles.borderRadius : 4}px`,
+    '& tfoot tr:last-child td:first-child': {
+      'border-bottom': 0,
+    },
+  }
```

Comment:
I feel like this part could be shortened with object restructuring. How about this?
```jsx
const defaultBorderStyle = {
  border: '1px solid #e0e0e0',
  borderRadius: 4,
}

const useStyles = ({ isBorder, 
borderStyles: {
    borderRadius, 
    borderWidth, 
    borderStyle, 
    borderColor
}, toolbar }) => makeStyles((theme) => {

  const containerBorderStyle = isBorder && {
    ...defaultBorderStyle,
    borderWidth,
    borderStyle,
    borderColor,
    borderRadius,
    '& tfoot tr:last-child td:first-child': {
      'border-bottom': 0,
    },
  }